public/css/list/redundancy-group-list-item.less

```diff
@@ -0,0 +1,20 @@
+.redundancy-group-list-item {
+  .caption {
+    display: flex;
+    justify-content: space-between;
+
+    .plugin-output {
+      .line-clamp(2);
+    }
+
+    .object-statistics {
+      ul {
+        display: flex;
```

Comment:
Why?

Reply:
TBH, this should be applied for `dependency-node-state-badges` to make all the state badges appear in a single row.

Reply:
They are in a single row, all state badges are. For me this rule didn't have any effect, which is why I'm asking.

Reply:
Ok, I should have been clear, this happens when you have both handled and unhandled state badges. And now that I have tested it again, it should be applied to the `dependency-node-state-badges` and all `ul` elements in the `dependency-node-state-badges`.

Reply:
Then I'm wondering why this needs to be specifically handled for dependency node state badges. They are not the only badges with handled/unhandled states which are shown in a single line.

Reply:
This is caused by `display: flex` on caption.

Reply:
Okay, seems plausible. But making something else flex because a parent is flex, cannot be the correct solution. Prevent the statistics from wrapping, that's all what's needed.